insights/parsers/tests/test_sys_vmbus.py

```diff
@@ -0,0 +1,81 @@
+import doctest
+import pytest
+from insights.parsers import sys_vmbus, SkipException
+from insights.tests import context_wrap
+
+
+BLANK = """
+""".strip()
+
+NO_RESULT = """
+ Id    Name                           State
+----------------------------------------------------
+""".strip()
+
+DEVICE_ID_1 = """
+{47505500-0001-0000-3130-444531444234}
+""".strip()
+
+CLASS_ID_1 = """
+{44c4f61d-4444-4400-9d52-802e27ede19f}
+""".strip()
+
+DEVICE_ID_2 = """
```

Comment:
This `DEVICE_ID_2` is not used in any test cases, better remove it or add a test for it.